packages/react-components/src/organisms/TeamCard.tsx

```diff
@@ -53,6 +55,11 @@ const overflowContentStyles = css({
 const teamMemberStyles = css({
   display: 'flex',
   alignItems: 'center',
+  color: lead.rgb,
+});
+
+const summaryStyles = css({
```

Comment:
`Paragraph` has an `accent` prop because changing color is very common and explicit in the design system

Reply:
Ah! Whoops. Thanks 👍